### ISSUE TITLE
AWS Image URL cors Issues only on Chrome browser

### ISSUE THREAD

**Issue by vparade** [ricardo-ch/react-easy-crop#272]
I am trying to get the crop image using below code
     
```
  const showCroppedImage = useCallback(async () => {
    try {
      const croppedImage = await getCroppedImg(URL, croppedAreaPixels)
      let imageFile = await getFileFromUrl(croppedImage, 'image.jpeg')
    } catch (e) {
      console.error(e)
    }
  }, [croppedAreaPixels])



const createImage = (url) =>
  new Promise((resolve, reject) => {
    const image = new Image()
    image.addEventListener('load', () => resolve(image))
    image.addEventListener('error', (error) => reject(error))
    image.setAttribute('crossOrigin', 'anonymous') // needed to avoid cross-origin issues on CodeSandbox
    image.src = url
  })
```
  
  My CORS policy for S3 bucket
```
  <?xml version="1.0" encoding="UTF-8"?>
[
    {
        "AllowedHeaders": [
            "*"
        ],
        "AllowedMethods": [
            "GET",
            "PUT",
            "POST",
            "DELETE"
        ],
        "AllowedOrigins": [
            "*"
        ],
        "ExposeHeaders": [],
        "MaxAgeSeconds": 30000
    }
]
```
  can u please let me know what's going wrong  **image.setAttribute('crossOrigin', 'anonymous')** 
  
  Error:
 Access to image at 'https://dev-usermanagement.s3-accelerate.amazonaws.com/----' from origin 'https://6fkxw.csb.app' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

**Comment by ValentinH** [comment]
I'm not familiar with AWS configuration but indeed you need it to accept serving images to a remote origin.

As the canvas cropping logic is not part of this library but only provided as an example, I'm going to close this issue. You should rather share your issue on Stackoverflow. 🙂

**Comment by lagroms** [comment]
I have the same problem here with AWS hosted images
